tests/errors/test__c.py

```diff
@@ -0,0 +1,3 @@
+from test_mod import i, test
+print(test(5.0))
+print(i)
```

Comment:
Works beautifully!
Here is the output:
```lp
$ lpython tests/errors/test__c.py --show-asr
semantic error: Type mismatch in annotation-assignment, the types must be compatible
 --> tests/errors/test_mod/test__a.py:3:1
  |
3 | i: Const[i32] = 1.23
  | ^               ^^^^ type mismatch ('i32' and 'f64')

semantic error: The module `test__a` cannot be loaded
 --> tests/errors/test_mod/__init__.py:1:1
  |
1 | from .test__a import i, test
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ imported here

semantic error: The module `__init__` cannot be loaded
 --> tests/errors/test__c.py:1:1
  |
1 | from test_mod import i, test
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ imported here


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that needs to be fixed).
```
Now, I'm working on addressing the @certik comments and simplifying things.

Reply:
Very good, thanks @Thirumalai-Shaktivel !